src/main/java/mil/nga/tiff/Rasters.java

```diff
@@ -306,7 +271,11 @@ public int getSamplesPerPixel() {
 	 * @return bits per sample
 	 */
 	public List<Integer> getBitsPerSample() {
```

Comment:
please add similar function public List<Integer> getSampleFormat() that returns list of sample formats (e.g.  TiffConstants.SAMPLE_FORMAT_UNSIGNED_INT)